server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/RangeAggregator.java

```diff
@@ -291,23 +290,8 @@ public static Aggregator build(
             cardinality,
             metadata
         );
-        Map<String, Object> filtersDebug = null;
```

Comment:
No more need to capture the filters debug - if we built the filters agg we kept it. If we didn't we don't have anything to keep. I suppose we could add a debug callback to the build process, but we already capture the queries it tries to build and those are all we use to decide whether or not to use filter by filter.